### ISSUE TITLE
Modeler GUI does not allow table values in a scn to be edited

### ISSUE THREAD

**Issue by michael-barrow** [sofa-framework/sofa#686]
I am unable to follow the pendulum tutorial at step 4 because it is not possible to edit position Vector 1/2 from the GUI

Expected outcome: 

Left click and typing of the number 2 in the X cell of the second position vector will update the X cell value from 1 to 2:

Actual outcome: It is not possible to edit any vector components in the GUI

Steps to reproduce: 
1 load examples/Tutorials/StepByStep/Pendulum/3_Pendulum.scn in the modeler application of v17.12 build. 
2 Double click on MechanicalObject Particles. Selcet Vector 1/2 tab of the GUI. 
3 Left click and type the number 2 in the X cell of the second position vector will update the X cell value from 0 to 1:

Below is the dialog box that should be visible when trying to edit the vector component.
![mechanicalobj scn](https://user-images.githubusercontent.com/20428997/41324725-19c31702-6e6b-11e8-98ca-268a77abe061.png)

______________________________________________________

Suggested labels:

**Comment by damienmarchal** [comment]
Hi @michael-barrow,

Thank you for reporting this issue with detailed explanation.

@guparan, @hugtalbot  I tried but wasn't able to reproduce the problem with a recent version of Sofa (not the 17.02), can anyone confirm (it was fixed or still an open issue).

**Comment by michael-barrow** [comment]
Dear @damienmarchal 

I had a typo in my original issue, it was 17.12. I have pulled from the stable branch of the GIT repository and built from source of 17.12 on Ubuntu 14.04. I have also reproduced this issue by building 17.02 from source using Ubuntu 18.04.

Although this issue is present in the "Modeler" binary, I find I am able to edit the vector fields if I load the .scn from the "RunSofa" binary.

Thank you for looking at this.

**Comment by michael-barrow** [comment]
Dear @damienmarchal 
It turns out that I can edit vector fields in Ubuntu 18.04. The relevant difference is the QT version. In 18.04 I am using QT v 5.9.5. My 14.04 machine has been torn down but if memory serves it was the minimum QT requested by CMake (version 5.5.x I believe).

Thank you.

**Comment by damienmarchal** [comment]
Dear @michael-barrow 

Normally the grayed out entries means that the corresponding data field has the "readOnly" flag set, so it is surprising the this behave differently in Modeller & runSofa. Maybe this is caused by some non-initialized variable somewhere in the Modeller, in runSofa or in the Data itself.  

When I will have time I will give a lookt at that to be sure it is not in the Data. If it turns out to be a problem specific to the Modeller well...as the Modeller is deprecated and should be replaced by runSofa2 (code it here https://github.com/sofa-framework/SofaQtQuick) I prefer to devote more time to this one :) 

Anyhow Michael, thanks again for your investigation, feel free also to update the issue if you notice similar problem. 

Regards,
Damien. 
EDIT: I changed my mind, so I changed the comment :)
